utils/debian_1_create.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo ================================
+echo === Generating debian folder ===
+echo ================================
+
+#ROOTDIR=buster
+ROOTDIR=$1
+if [ -z "$ROOTDIR" ]
+then
+echo "Select a distribution" ||
+exit 1
+fi
+
+cd out/linux64/build
+
+sudo debootstrap --arch amd64 --no-check-gpg $ROOTDIR $ROOTDIR http://http.debian.net/debian/
```

Comment:
I'd suggest using a HTTPS-base mirror instead, see e.g. a list [here](https://gist.github.com/eighthave/7285154#gistcomment-2820716)

Reply:
Ok I selected https://deb.debian.org/debian and leave it to users to find a faster mirror near them.

This server behaves strange if you use your browser and redirects to http mirror, but seems to work fine with debootstrap.